src/cdk/overlay/overlay-config.ts

```diff
@@ -59,11 +59,6 @@ export class OverlayConfig {
    */
   disposeOnNavigation?: boolean = false;
 
-  /**
-   * Array of HTML elements clicking on which should not be considered as outside click
-   */
-  excludeFromOutsideClick?: HTMLElement[] = [];
```

Comment:
I think we talked about having this be a predicate function instead of an array of elements.

Reply:
@jelbourn we later agreed that we should just emit on the `_outsidePointerEvents` subject and have the user decide correct?

Reply:
Yeah, when Andy and I follow-up on the discussion in the team meeting, I remembered that the `outsidePointerEvents` stream is, well, a stream that people can filter themselves however they want, so adding a predication would literally just be adding a call to `filter`, so calling the API is the same amount of work as adding a `filter` yourself.

Reply:
@jelbourn thinking about it now, adding a predicate can stop overlays added earlier from being processed since the loop would just break based on the predicates return. I can implement the menu with either approach but I'm not sure if there's some edge cases where we may want an early break.